help/versions.md

```diff
@@ -1,59 +1,95 @@
+
+Please see the [references and citations help page](./faq/references) for details on citing works in arXiv.
+
+
+## What constitutes a version?
 
 A version will be created once a [replacement](replace) or
-[withdrawal](withdraw) submission is made public. The *Submission
-history* section of the Abstract page indicates the version number being
-viewed, as well as all available versions.
+[withdrawal](withdraw) submission is made public. The _Submission history_ section of the Abstract page indicates the version number being viewed, as well as all available versions.
+
+Replacements will contain the full text of the article. Withdrawals will display the reason for withdrawal entered as a comment, without any content download options for that version.
+
+
+## Situations when arXiv typically requires versioning
+
+Below are cases where arXiv authors are asked to version their work under a single arXiv identifier rather than to submit related works as new arXiv identifier.
+
+### Articles in stages of the publication process
+
+Version in many people’s mind represent various stages along publication status:
+
+- Submitted version
+- Revised version
+- Accepted version
+- Published version
+  - Includes journal, conferences or other venues. Even if each has their own version of record, DOI, and citation.
+
+Note that during these stages a paper may have changes to the Title as well as to the Author list that are different in each version. These changes can be noted in the metadata.
+ 
+### Related works with various content
+
+Versions in arXiv may also represent a set of related works that includes various content types. There can be versions of the work that includes:
+
+- Supplemental content (appendix, data, videos, code)
+- Erratum either appended or corrected within the text
+- Expanded versions
+- Short or condensed version for different reader groups
+- Full version
+ 
+ 
+### Other cases
+ 
+#### Famous proofs
+
+Versions in arXiv are also used in cases of an author making various attempts to work through a famous proof, even if the content of the work changes entirely between versions.
+ 
+#### Splitting or Merging works
+
+*Splitting* - A long work that is posted on arXiv may be split into two or more shorter works. In arXiv the current practice is to replace the previous long version with one of the new shorter works. Then the other shorter works can be posted as new arXiv papers with a new arXiv identifier.
+
+*Merging* - Two or more short works that are posted on arXiv may be merged into a longer work. In arXiv the current practice is the longer work is a replacement of one of the existing short works. Then the other short work is withdrawn with a note that it has been superceded by the other arXiv paper, including the arXiv identifier in the note so that readers can link between the works.

```

Comment:
```suggestion
*Merging* - Two or more short works that are posted on arXiv may be merged into a longer work. In arXiv the current practice is to replace one of the existing short works with the longer work. Then the other short work is withdrawn with a note that it has been superceded by the longer arXiv paper. The arXiv identifier of the longer paper is included in the note so that readers can link between the works.

```